src/cli/near.rs

```diff
@@ -70,13 +82,61 @@ pub enum ReadCommand {
         #[clap(subcommand)]
         contract_call: crate::cli::solidity::Solidity,
     },
+    // get nep141_from_erc20
     GetBridgedNep141 {
         erc_20_address_hex: String,
     },
     GetAuroraErc20 {
         nep_141_account: String,
     },
     GetEngineBridgeProver,
+    // get_chain_id
+    GetChainId,
+    // get_upgrade_index
+    GetUpgradeIndex,
+    // get_block_hash
+    GetBlockHash,
```

Comment:
`get_block_hash` has the [height as an input](https://github.com/aurora-is-near/aurora-engine/blob/master/engine/src/lib.rs#L424).

Reply:
I see engine reads as a borsh input, but I was not able to look for its argument type. Is it ok to just include input(block_number) in input byte array?

Reply:
I added number as argument with serialized little-endian byte array, but `near-jsonrpc` shows this following error.

```
Error: NearRpc(TransportError(RecvError(ResponseParseError(ResultParseError(Error("data did not match any variant of untagged enum QueryResponseKind", line: 0, column: 0))))))
```